net/wireshark22/Portfile

```diff
@@ -4,14 +4,20 @@ PortSystem          1.0
 PortGroup           cmake 1.0
 
 name                wireshark22
-version             2.2.10
-revision            6
+version             2.2.17
+revision            0
 categories          net
 license             {GPL-2 GPL-3}
 maintainers         {darkart.com:opendarwin.org @ghosthound}
 description         Graphical network analyzer and capture tool
 homepage            https://www.wireshark.org/
-platforms           darwin
+# macOS 10.12+ are supported with Wireshark 3.x/4.x; 10.11 and below are limited to 2.x.
```

Comment:
Does "macOS 10.12+" mean "any macOS version from 10.12 to latest" or "any macOS 10.12.x version"?  I think the latter, and if so the comment should be adjusted to be clearer that we're only talking about macOS 10.12.x here.

Reply:
@ghosthound I actually meant 10.12 and later ones. (Strictly speaking, macos-13 does not support 3.x, according to upstream, but I guess this detail is not relevant to 2.x ports.)
You are right, it is unclear. Perhaps “10.12 and later” will do?

Reply:
If the meaning is "any version of macOS from 10.12 to current" then I think "10.12 and later" works.

And thanks for doing the work on this!

Reply:
Then should be good now, I guess. Thank you!